server/src/main/java/com/cloud/resource/ResourceManagerImpl.java

```diff
@@ -1840,9 +1840,9 @@ private void handleAutoEnableDisableKVMHost(boolean autoEnableDisableKVMSetting,
                 _hostDetailsDao.update(hostDetail.getId(), hostDetail);
             } else if (!isUpdateFromHostHealthCheck && hostDetail != null &&
                     Boolean.parseBoolean(hostDetail.getValue()) && resourceEvent == ResourceState.Event.Disable) {
-                logger.info(String.format("The setting %s is enabled but the host %s is manually set into %s state," +
+                logger.info("The setting {} is enabled but the host {} is manually set into {} state," +
```

Comment:
And this file also?

Reply:
What's the issue with this ? this was being reported by sonar - so I fixed it.

Reply:
Not an issue, I just see it redundant

Reply:
I meant using `String.format` or `{}` inside the logger.info has the same effect, anyways I'm fine with it

Reply:
String.format() has been removed though.